tools/benchmark_tool/openvino/tools/benchmark/main.py

```diff
@@ -270,11 +268,16 @@ def set_throughput_streams():
             # --------------------- 4. Read the Intermediate Representation of the network -----------------------------
             next_step()
 
+            logger.info("Loading network files")
+
             start_time = datetime.utcnow()
             model = benchmark.read_model(args.path_to_model)
             topology_name = model.get_name()
             duration_ms = f"{(datetime.utcnow() - start_time).total_seconds() * 1000:.2f}"
-            logger.info(f"Read model took {duration_ms} ms")
+            logger.info(f"Read network took {duration_ms} ms")
```

Comment:
rollback all model to network related changes.

Reply:
Rolled back!